resources/service.rb

```diff
@@ -1,93 +1,94 @@
 include Iptables::Cookbook::Helpers
 
-property :ip_version, Symbol,
-          equal_to: %i(ipv4 ipv6),
-          default: :ipv4,
-          description: 'The IP version, 4 or 6'
-
-property :sysconfig, Hash,
-          default: lazy { get_sysconfig(ip_version) },
-          description: 'The sysconfig settings'
-
-property :service_name, String,
-          default: lazy { get_service_name(ip_version) },
-          description: 'Name of the iptables services'
+property :config_file, String,
+          default: lazy { default_iptables_rules_file(ip_version) },
+          description: 'The full path to find the rules on disk'
 
-property :sysconfig_file_mode, String,
-          default: '0600',
-          description: 'Permissions on the saved sysconfig file'
+property :owner, String,
+          default: lazy { default_iptables_file_owner },
+          description: 'Permissions on the saved output file'
 
-property :file_mode, String,
-          default: '0644',
-          description: 'Permissions on the saved rules file'
+property :group, String,
+          default: lazy { default_iptables_file_group },
+          description: 'Permissions on the saved output file'
 
-property :source_template, String,
-          default: 'iptables-config.erb',
-          description: 'Source template to use to create the sysconfig file'
+property :mode, String,
+          default: lazy { default_iptables_file_mode },
```

Comment:
Splitting these strings out into helper libraries seems like over DRYing the codebase. If you can avoid lazy evals you always should

Reply:
I'd carried that over from the sous cookbooks. I thought about going resource partials instead but was a bit hesistant to cap it at >= 16.